babel.config.js

```diff
@@ -12,17 +12,13 @@ module.exports = (api) => {
         '@babel/preset-env',
         {
           targets: {
-            node: 'current',
+            // TODO: can we get this to read from package.json somehow..?
+            node: '12',
           },
         },
       ],
-      '@babel/preset-react',
-    ],
-    overrides: [
-      {
-        test: /\.tsx?$/,
-        presets: ['@babel/preset-typescript'],
-      },
+      '@babel/preset-typescript',
     ],
+    plugins: ['@babel/plugin-proposal-class-properties'],
```

Comment:
Q: why do we need to add this additional plugin on top of `@babel/preset-typescript`? Are there any more we need?